pkgs/sketch_pad/lib/main.dart

```diff
@@ -67,6 +67,7 @@ class _DartPadAppState extends State<DartPadApp> {
   @override
   void initState() {
     router.routeInformationProvider.addListener(_setTheme);
+    _setTheme();
     super.initState();
```

Comment:
I think we want the first line of this method to be the call to super.initState().